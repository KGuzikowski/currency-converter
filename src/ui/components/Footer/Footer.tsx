import React from 'react'
import { Theme, createStyles, makeStyles } from '@material-ui/core/styles'
import Typography from '@material-ui/core/Typography'
import Link from '@material-ui/core/Link'

const useStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            padding: theme.spacing(2),
            backgroundColor: theme.palette.primary.main,
            color: 'white',
            textAlign: 'center'
        },
        name: {
            color: theme.palette.secondary.main,
            fontWeight: 'bold'
        }
    })
)

const Footer = () => {
    const classes = useStyles()
    return (
        <div className={classes.root}>
            <Typography variant='body1'>
                Let's convert some currencies! App made for recruitment purposes by <span> </span>
                <Link href='https://github.com/KGuzikowski' target='_blank' rel='noopener' className={classes.name}>
                    Karol Guzikowski
                </Link>.
            </Typography>
        </div>
    )
}

export default Footer