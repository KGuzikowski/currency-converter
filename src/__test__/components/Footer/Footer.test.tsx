import React from 'react'
import { create } from 'react-test-renderer'
import Footer from '../../../ui/components/Footer/Footer'

it('renders Footer component correctly', () => {
    const component = create(<Footer />)
    expect(component.toJSON()).toMatchSnapshot()
})