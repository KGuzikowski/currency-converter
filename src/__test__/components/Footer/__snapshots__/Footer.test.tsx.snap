// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer component correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    Let's convert some currencies! App made for recruitment purposes by 
    <span>
       
    </span>
    <a
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-name-2 MuiTypography-colorPrimary"
      href="https://github.com/KGuzikowski"
      onBlur={[Function]}
      onFocus={[Function]}
      rel="noopener"
      target="_blank"
    >
      Karol Guzikowski
    </a>
    .
  </p>
</div>
`;
